tools/alerts/create_alerts.py

```diff
@@ -5,13 +5,18 @@
 import os
 import re
 from collections import defaultdict
+from datetime import datetime, time, timedelta, timezone
 from difflib import SequenceMatcher
 from typing import Any, Dict, List, Set, Tuple
 
 import requests
 from setuptools import distutils  # type: ignore[import]
 
-from tools.github.github_utils import gh_fetch_commit
+from tools.github.github_utils import gh_fetch_commit  # type: ignore[import]
+from tools.stats.rockset_utils import run_rockset_query
+
+MINIMUM_STRICT_LAG = 3 * 60 * 60  # 2 hours
+MAXIMUM_STRICT_LAG = 6 * 60 * 60  # 6 hours
```

Comment:
I guess we can tweak this later if needed, 6 hours seems a bit noisy.  I'm thinking like 12 hours, but let's see

Reply:
Never mind, the threshold on HUD metric page is 6 hours, so let's stick with that as you have here